besu/src/main/java/org/hyperledger/besu/cli/custom/RpcAuthFileValidator.java

```diff
@@ -97,7 +99,20 @@ private static boolean verifyAllEntriesHaveValues(final TomlParseResult tomlPars
         .dottedKeySet()
         .parallelStream()
         .filter(keySet -> !keySet.contains("password"))
-        .allMatch(dottedKey -> verifyArray(dottedKey, tomlParseResult));
+        .allMatch(dottedKey -> verifyEntry(tomlParseResult, dottedKey));
+  }
+
+  private static boolean verifyEntry(final TomlParseResult tomlParseResult, final String key) {
+    if (key.endsWith(PRIVACY_PUBLIC_KEY)) {
+      return verifyString(key, tomlParseResult);
+    } else {
+      return verifyArray(key, tomlParseResult);
+    }
+  }
+
+  private static boolean verifyString(final String key, final TomlParseResult tomlParseResult) {
```

Comment:
nit; is there a reason the ordering of the input parameters is inconsistent between `verifyEntry` and `verifyString`?

Reply:
nope, nice catch.
done